helm/config/nginx.conf

```diff
@@ -18,6 +18,16 @@ http {
         endpoint {{ .Values.opentelemetry.exporterHost }}:{{ .Values.opentelemetry.exporterPort }};
     }
 
+    map $server_protocol $otel_network_protocol_name {
+        ~^(.+)/.*$  $1;
+        default     "http";
+    }
+
+    map $server_protocol $otel_network_protocol_version {
+        ~^.+/(.+)$  $1;
+        default     "1.1";
+    }
+
```

Comment:
can you explain please?

Reply:
yes i didnt understand the dark magic of   ~^.+/(.+)$

Reply:
I simplified the regex.